### ISSUE TITLE
custom data html attributes

### ISSUE THREAD

**Issue by Tsur** [featurist/plastiq#42]
Hi! and thank you all for working on this great framework.  

I'm trying it out to set a custom data and contenteditable attribute but not sure if i'm doing something wrong. This is not working for me:

```js
h('span', { 'data-type':'somethingHere', 'contenteditable': true }, 'name: ', model.name);
```
Anyhow, I'll check the source code to check what's going on.

Thank you very much!

**Comment by joshski** [comment]
Hi @Tsur,

Non-standard HTML attributes should be added to an `attributes` object, like this:

```JavaScript
var plastiq = require("plastiq");
var h = plastiq.html;

function render(model) {
  return h(".editable", { attributes: { contentEditable: true } }, "Edit me");
}

plastiq.append(document.body, render, {});
```

...however, beware that setting contenteditable to true may not have the effect you're looking for! That is, I don't think plastiq hooks into any contenteditable events to support any kind of updating of the model.

**Comment by refractalize** [comment]
Hi @Tsur,

For these you want to use `contentEditable: true` and `dataset: {type: 'somethingHere'}`.

@joshski turns out `contentEditable` does seem to work outside of `attributes: {}`, just been using it now by chance!

virtual-dom uses JavaScript names for things like `className` and `contentEditable` but plastiq prefers to stay closer to HTML since that's much more familiar to people. I tend to change them as I bump into them so I'll get these changed (continuing to allow the JavaScript names too of course.)

**Comment by msegundo** [comment]
Thanks!! It works.

**Comment by refractalize** [comment]
fixed in 96f949189d95eac80613501ae26e4959d34188a6, and released in v1.17.0

**Comment by Tsur** [comment]
It worked on the new release, great! thank you!
